src/renderer/components/ReaderApp.tsx

```diff
@@ -151,10 +151,10 @@ const queryParams = getURLQueryParams();
 
 // tslint:disable-next-line:no-string-literal
 const publicationJsonUrl = queryParams["pub"];
-const publicationJsonUrl_ = publicationJsonUrl.startsWith(READIUM2_ELECTRON_HTTP_PROTOCOL) ?
+const newPublicationJsonUrl = publicationJsonUrl.startsWith(READIUM2_ELECTRON_HTTP_PROTOCOL) ?
```

Comment:
This change is incorrect. It is very important that `publicationJsonUrl` remains exactly as passed in the `pub` querystring URL parameter for the `index_reader.html` renderer process. Then, `newPublicationJsonUrl_` (with the underscore suffix) is only used to compute the publication path / filename, by decoding its base64 parameter.